configuration.rst

```diff
@@ -914,6 +914,80 @@ parameters at once by type-hinting any of its constructor arguments with the
         }
     }
 
+.. _config-config-builder:
+
+Using PHP ConfigBuilders
+------------------------
+
+Writing PHP config is sometimes difficult because you end up with large nested arrays
+and you have no help from your favorite IDE. A way to address this is to use "ConfigBuilders".
+They are objects that will help you build these arrays.
+
+.. versionadded:: 5.3
+
+    The "ConfigBuilders" was added in Symfony 5.3 as an :doc:`experimental feature </contributing/code/experimental>`.
+
+The ConfigBuilders are automatically generated in your ``kernel.build_dir`` for
+every bundle. By convention they all live in the namespace ``Symfony\Config``.
+
+.. configuration-block::
+
+    .. code-block:: php
```

Comment:
I think the `php` example should not be the first tab, and people should see the new `ConfigBuilder` by default.